src/coreComponents/physicsSolvers/multiphysics/MultiResolutionHFSolver.cpp

```diff
@@ -0,0 +1,438 @@
+  //  .setApplyDefaultValue(0.0)
+  //  .setPlotLevel( PlotLevel::NOPLOT )
+  //  .setDescription("").;
+
+  //     //register fix displacement values
+  //     NodeManager & nodes = mesh.getNodeManager();
+  //     nodes.registerWrapper< array1d<localIndex> >(SolidMechanicsLagrangianFEM::viewKeyStruct::nameChosenString() )
+  //  .setApplyDefaultValue(0.0)
+  //  .setPlotLevel( PlotLevel::NOPLOT )
+  //  .setDescription("").;
+
+  //     //register fix damage nodes
+  //     NodeManager & nodes = mesh.getNodeManager();
+  //     nodes.registerWrapper< array1d<localIndex> >(PhaseFieldDamageFEM::viewKeyStruct::nameChosenString() )
+  //  .setApplyDefaultValue(0.0)
+  //  .setPlotLevel( PlotLevel::NOPLOT )
+  //  .setDescription("").;
+
+
+  // });
+  // Andre - 03/14 - keep this function empty for later
+  GEOSX_UNUSED_VAR( MeshBodies );
+}
+
+MultiResolutionHFSolver::~MultiResolutionHFSolver()
+{
+  // TODO Auto-generated destructor stub
+}
+
+void MultiResolutionHFSolver::implicitStepSetup( real64 const & time_n,
+                                                 real64 const & dt,
+                                                 DomainPartition & domain )
+{
+  //Andre - 14/03/22 - can we just call these functions without doing the boundary condition operations first?
+  m_baseSolver->implicitStepSetup( time_n, dt, domain );
+  m_patchSolver->implicitStepSetup( time_n, dt, domain );
+}
+
+void MultiResolutionHFSolver::postProcessInput()
+{
+  SolverBase::postProcessInput();
+  m_baseSolver = &this->getParent().getGroup< SolidMechanicsEmbeddedFractures >( m_baseSolverName );
+  m_patchSolver = &this->getParent().getGroup< PhaseFieldFractureSolver >( m_patchSolverName );
+}
+
+void MultiResolutionHFSolver::initializePostInitialConditionsPreSubGroups()
+{}
+
+void MultiResolutionHFSolver::resetStateToBeginningOfStep( DomainPartition & domain )
+{
+  m_baseSolver->resetStateToBeginningOfStep( domain );
+  m_patchSolver->resetStateToBeginningOfStep( domain );
+}
+
+real64 MultiResolutionHFSolver::solverStep( real64 const & time_n,
+                                            real64 const & dt,
+                                            int const cycleNumber,
+                                            DomainPartition & domain )
+{
+  real64 dtReturn = dt;
+  dtReturn = splitOperatorStep( time_n, dt, cycleNumber, domain );
+  return dtReturn;
+}
+
+//in the initial step and whenever the subproblem mesh moves, we must update the node map
+// void MultiResolutionHFSolver::updateNodeMaps( MeshLevel const & base,
+//            MeshLevel const & patch )
+// {
+//   // get list of nodes on the boundary of the patch
+//   NodeManager const & patchNodeManager = patch.getNodeManager();
+//   NodeManager const & baseNodeManager = base.getNodeManager();
+//   SortedArrayView< localIndex const > const patchExternalSet = patchNodeManager.externalSet();
```

Comment:
I have filled the `m_isExternal` in the `nodeManager`. I think you need to use that one and not this `externalSet()`.